index.bs

```diff
@@ -4590,3 +4777,14 @@ Possible mitigations:
     presence from the reporting origin. Compared to the previous mitigation, the
     proxy server could itself handle the [=event-level report/trigger priority=]
     functionality, at the cost of increased complexity in the proxy.
+
+## Attribution Scope ## {#attribution-scope}
+
+*This section is non-normative.*
+
+Abusive actors can exploit the attribution scope feature by registering ghost clicks instead of views. During source registration, they can redirect to themselves and register [=attribution source/attribution scope limit=]-1 clicks. This allows them to choose more complex configurations (more bits) for their ghost registrations.
+
+To prevent this, we can limit the number of source registrations per adtech platform per registration for sources that use the attribution scope feature.
+
+Proposed mitigation: Limit to 1 click registration per reporting origin, per source registration (applicable only when the attribution scope feature is enabled).
```

Comment:
"when the attribution scope feature is enabled" is a bit vague. Would it be more precise to enable this when the source has non-empty scopes?

Also, does this mean that the reporting origin can only register 1 navigation source period, or only 1 navigation source with non-empty scopes?

Reply:
only 1 navigation source with non-empty scopes.